adding font-optical-sizing example

chrisdavidmills opened the pull request:
Note that this currently doesn't work properly, so consider this a WIP PR.

@schalkneethling is gonna help me investigate it, as discussed over mail. Thanks!

schalkneethling commented on live-examples/css-examples/fonts/font-optical-sizing.html:
```diff
@@ -0,0 +1,25 @@
+
+<div id="output" class="output hidden">
+    <section id="default-example">
+        <p>Optical sizing example.</p>
```
@chrisdavidmills You are missing an id of example-element here. With that, this will work. It seems pretty unclear what the browser support for this is though. It seems to be in Safari and in an insiders version of Edge. Seems not all the things work in Chrome, and it is behind a flag in Firefox although, not sure what that flag is.

Still great to have an example for when it is supported though.

schalkneethling commented on live-examples/css-examples/fonts/font-optical-sizing.html:
```diff
@@ -0,0 +1,25 @@
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
```
Nit: Code style: Indent this and the previous block with two spaces.

schalkneethling commented on live-examples/css-examples/fonts/font-optical-sizing.html:
```diff
@@ -0,0 +1,25 @@
+<section id="example-choice-list" class="example-choice-list" data-property="font-size">
+
+
```
Nit: Remove two empty lines.

chrisdavidmills commented:
Thanks @schalkneethling ! So, I've removed the extraneous file and added the missing ID. I tried building it and testing locally, and the example now works perfectly.

But, on commiting, one of the tests still fails.

I've just used `--no-verify` to commit this.

This is currently supported in Chrome, Saf, and Firefox behind the `layout.css.font-variations.enabled` pref

schalkneethling commented on live-examples/css-examples/fonts/font-optical-sizing.html:
```diff
@@ -0,0 +1,25 @@
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
```
Nit: Remove one empty line.

schalkneethling commented on live-examples/css-examples/fonts/font-optical-sizing.html:
```diff
@@ -0,0 +1,25 @@
+
+</section>
+
+
```
Nit: Remove one empty line.

chrisdavidmills commented:
Done!
